demo-reviewnb.ipynb

```diff
@@ -94,19 +94,19 @@
   },
```

Comment:
<p>Seems like you can't comment on lines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/joelostblom/mds-demo/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>